src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandOddsRatio.java

```diff
@@ -20,37 +20,100 @@
+ * incorrect evaluation of the amount of evidence observed for one allele vs. the other. The StrandOddsRatio annotation
+ * is one of several methods that aims to evaluate whether there is strand bias in the data. It is an updated form of
+ * the <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.php">Fisher Strand Test</a>
+ * that is better at taking into account large amounts of data in high coverage situations. It is used to determine if
+ * there is strand bias between forward and reverse strands for the reference or alternate allele(s).</p>
  *
  * <h3>Statistical notes</h3>
- * <p> Odds Ratios in the 2x2 contingency table below are</p>
  *
- * $$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$
- *
- * <p>and its inverse:</p>
+ * <p>The following 2x2 contingency table gives the notation for allele support and strand orientation.</p>
  *
  * <table>
- *      <tr><td>&nbsp;</td><td>+ strand </td><td>- strand</td></tr>
- *      <tr><td>REF;</td><td>X[0][0]</td><td>X[0][1]</td></tr>
- *      <tr><td>ALT;</td><td>X[1][0]</td><td>X[1][1]</td></tr>
+ * <tr><th>&nbsp;</th><th>+ strand&nbsp;&nbsp;&nbsp;</th><th>- strand&nbsp;&nbsp;&nbsp;</th></tr>
+ * <tr><th>REF&nbsp;&nbsp;&nbsp;</th><td>X[0][0]</td><td>X[0][1]</td></tr>
+ * <tr><th>ALT&nbsp;&nbsp;&nbsp;</th><td>X[1][0]</td><td>X[1][1]</td></tr>
  * </table>
  *
- * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT (the sum makes it symmetric). A high value is indicative of large difference where one entry is very small compared to the others. A scale factor of refRatio/altRatio where</p>
+ * <p>We can then represent the Odds Ratios with the equation:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$" border="0"/>
+ *
+ * <p>and its inverse:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ \frac{1}{R} = \frac{X[0][1] * X[1][0]}{X[0][0] * X[1][1]} $$" border="0"/>
  *
- * $$ refRatio = \frac{max(X[0][0], X[0][1])}{min(X[0][0], X[0][1} $$
+ * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT. The sum makes it symmetric.
+ * A high value is indicative of large difference where one entry is very small compared to the others. A scale factor
+ * of refRatio/altRatio where</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ refRatio = \frac{min(X[0][0], X[0][1])}{max(X[0][0], X[0][1])} $$" border="0"/>
  *
  * <p>and </p>
  *
- * $$ altRatio = \frac{max(X[1][0], X[1][1])}{min(X[1][0], X[1][1]} $$
+ * <img src="http://latex.codecogs.com/svg.latex?$$ altRatio = \frac{min(X[1][0], X[1][1])}{max(X[1][0], X[1][1])} $$" border="0"/>
+ *
+ * <p>ensures that the annotation value is large only. The final SOR annotation is given in natural log space.</p>
```

Comment:
What does it mean that the scale factor "ensures that the annotation value is large only"?

Reply:
I assume the old text comes from a developer who is speaking math lingo that means something to other math people. Thought you approved of these earlier?